todo-app/src/components/todoList.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from "react";
+import TaskItem from "./taskItem.js";
+import AddingContainer from "./addingContainer.js"
+
+export let todoArray = [
+    { description: 'Get out of bed', deadLine: 'Wed Sep 13 2017' },
+    { description: 'Brush teeth', deadLine: 'Thu Sep 14 2017' },
+    { description: 'Eat breakfast', deadLine: 'Fri Sep 15 2017' }
+]
+
+class TodoList extends Component {
+    constructor(props) {
+        super(props)
+        this.state = {
+            tasks: todoArray
+        }
+    }
+   
+    addingTask = (description, deadLine) => {
+        this.setState((previousState) => ({
```

Comment:
Careful here. If I'm not mistaken, using this function argument _does not_ enable shallow merging of state. It's also a lot less commonly used.